src/compiler/perfLogger.ts

```diff
@@ -0,0 +1,81 @@
+/* @internal */
+namespace ts {
+    export type PerfLogger = typeof import("@microsoft/typescript-etw"); // tslint:disable-line:no-implicit-dependencies
+
+    export class NullLogger implements PerfLogger {

```

Comment:
Instead of making this class make it a object with all the methods = noop that is
```ts
export const nullLogger: PerfLogger = { 
 logEvent: noop,
logErrEvent: noop// and so on
};
```

Reply:
Done.